src/components/EcommerceCard.js

```diff
@@ -1,28 +1,49 @@
+import { useState } from 'react';
 import { faHeart } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { useAuth } from '../context/auth-context';
+import { addToWishList, removeFromWishList } from '../util/product-request';
 import Card from './Card';
 import './EcommerceCard.css';
+import { useProduct } from '../context/product-context';
+import loader from '../assets/loaders/loader.gif';
```

Comment:
Separate the module import with local import will increase the readability
Ex:
//All module imports at the start
import axios from "axios";

//All local import
import Card from './Card';

Reply:
The code base is following the same pattern as you mentioned,
Imports from the external libraries and then local imports